pom.xml

```diff
@@ -79,55 +127,73 @@
 			<groupId>javax.jcr</groupId>
 			<artifactId>jcr</artifactId>
 			<version>2.0</version>
-            <optional>true</optional>
+			<optional>true</optional>
+			<scope>provided</scope>
 		</dependency>
 		<dependency>
 			<groupId>org.slf4j</groupId>
 			<artifactId>slf4j-api</artifactId>
+			<scope>provided</scope>
 		</dependency>
 		<dependency>
-            <!-- A recent version is required to use Sling mocks in tests, if that's a problem we'll need to change them -->
```

Comment:
Please revert this whitespace-only change, it makes checking the history more difficult.